CMD/CommandLineSettings.cs

```diff
@@ -32,7 +32,12 @@ public class CommandLineSettings
 
         [Option('v', HelpText = "Runs a small test search using a database and yeast data file included with this MetaMorpheus installation")]
         public bool RunMicroVignette { get; set; }
-        
+
+        [Option("verbosity", Default = VerbosityType.normal, HelpText = "Determines how much text is written. Options are no output ('none'), minimal output and errors  ('minimal'), or normal ('normal')")]
```

Comment:
Typically, this is `-v` in commandline programs. Consider changing the test flag to `--test`. Is the default showed in the help menu? If not, maybe add that to the description.

Reply:
thx, i've noticed that too since implementing the microvignette. i've made -v refer to verbosity and --test runs the microvignette.

the default verbosity is displayed when --help is called

Reply:
Also, since we're here, I think it'd be helpful to note that the databases, spectra, and tomls are space-delimited in their descriptions (https://github.com/smith-chem-wisc/MetaMorpheus/issues/1593)

Reply:
will do